pkg/sql/catalog/lease/count.go

```diff
@@ -35,14 +51,16 @@ func CountLeases(
 		)
 	}
 
+	singleRegionClause, singleRegionParam := getSingleRegionClause(isMultiRegion)
 	stmt := fmt.Sprintf(`SELECT count(1) FROM system.public.lease AS OF SYSTEM TIME '%s' WHERE `,
 		at.AsOfSystemTime()) +
-		strings.Join(whereClauses, " OR ")
+		strings.Join(whereClauses, " OR ") +
+		singleRegionClause
```

Comment:
I think this is incorrect. See my comment here: https://github.com/cockroachdb/cockroach/pull/119466#pullrequestreview-1903819379